packages/@aws-cdk/aws-ec2/lib/port.ts

```diff
@@ -0,0 +1,201 @@
+import { Token } from '@aws-cdk/cdk';
+
+/**
+ * Protocol for use in Connection Rules
+ */
+export enum Protocol {
+  ALL = '-1',
+  TCP = 'tcp',
+  UDP = 'udp',
+  ICMP = 'icmp',
+  ICMPV6 = '58',
+}
+
+/**
+ * Properties to create a port range
+ */
+export interface PortProps {
+  /**
+   * The protocol for the range
+   */
+  readonly ipProtocol: Protocol;
+
+  /**
+   * The starting port for the range
+   *
+   * @default - Not included in the rule
+   */
+  readonly fromPort?: number;
+
+  /**
+   * The ending port for the range
+   *
+   * @default - Not included in the rule
+   */
+  readonly toPort?: number;
+
+  /**
+   * String representation for this object
+   */
+  readonly stringRepresentation: string;
+}
+
+/**
+ * Interface for classes that provide the connection-specification parts of a security group rule
+ */
+export class Port {
+  /**
+   * A single TCP port
+   */
+  public static tcpPort(port: number): Port {
```

Comment:
Wouldn't `Port.tcp(12)` `Port.tcpAll()` and `Port.tcpRange(..)` etc be sufficient? No need to repeat "port"

Reply:
Yeah you're right.